test/test.js

```diff
@@ -6,13 +6,14 @@ const toArray = require('stream-to-array');
 
 const postgresCSVLog = require('../index');
 
-it('should parse log lines correctly', (done) => {
+it('should correctly parse error log lines', (done) => {
   stream = fs.createReadStream('test/test_log.csv').pipe(postgresCSVLog());
   toArray(stream, (err, arr) => {
     if (err) {
       done(err);
     } else {
-      _.sortBy(arr, 'log_time').should.eql([
+      const errorRecord = _.sortBy(arr, 'log_time')[0];
```

Comment:
This seems like a weird pattern. What's wrong with the existing thing? If we want to isolate each parsing case, we should be splitting up the input rather than output?

Reply:
It's clearer to have separate test cases for each type of thing we're parsing.

What exactly do you mean by splitting the input here?

Reply:
Right now, each test case actually processes the same thing, but different parts of the result is checked. As someone reading the tests, it makes it less obvious what each test actually does. If we were to split up the input into test cases 1,2,3, such that each test processes one of the parts, it would be a lot more understandable. However, I suppose we will lose coverage that tests how the lib handles a stream of a few inputs.

If separate test cases is important to you, just add comment that explain in each test case what we are processing the whole file, but we want to specifically check how line X is handled